README.md

```diff
@@ -1,5 +1,7 @@
 # Sales-Lead-Qualifier Project (AMOS WS 2023/24)
 
+![Screenshot 2023-10-24 at 19 48 33](https://github.com/amosproj/amos2023ws06-sales-lead-qualifier/assets/45459787/003c9302-760f-4ca7-8c7a-12be99886715)
```

Comment:
It would be nicer to have a proper alt-text here.
Also, why are you not using the `team-logo.png` directly? Is the transparent background a problem?

Reply:
If I put the ones with transparent background , there will be people experiencing the same issue as you did yesterday. On dark mode, logo  in black will not  be visible and on light mode vice versa.

Reply:
Makes sense

Reply:
Take a look at this: https://github.blog/changelog/2021-11-24-specify-theme-context-for-images-in-markdown/ 🥳